src/player.js

```diff
@@ -259,14 +259,14 @@ export class Player extends Phaser.GameObjects.Container {
             this.body.setVelocityY(this.scene.gamepad.joystick.properties.y / 100 * speed);
         }
 
-        if (this.scene.cursors.left.isDown) {
+        if (this.scene.cursors.left.isDown && this.body.position.x > 10) {
             this.body.setVelocityX(-100);
-        } else if (this.scene.cursors.right.isDown) {
+        } else if (this.scene.cursors.right.isDown && this.body.position.x < 5 * 32 * 16 - 25) {
```

Comment:
this implementation is a bit problematic:
- all values are hardcoded, so it's hard to change map size
- it doesn't work with virtual gamepad

I think there is built-in support for this in Phaser 3 physics however:
https://www.html5gamedevs.com/topic/38972-solved-issue-with-world-bounds-collision/